src/main/java/org/bigredbands/mb/views/FieldStyle.java

```diff
@@ -0,0 +1,92 @@
+package org.bigredbands.mb.views;
+
+import java.awt.Color;
+
+public class FieldStyle {
+    // ****************************
+    // ***** Field properties *****
+    // ****************************
+    private float fieldNumberSize;
```

Comment:
I know we don't have an official style guide for this project, but general Java style (and what is used elsewhere in the project) have getters named `getX()` and setters named `setY()`.  Can you update the getters and setters in this file to match?

Reply:
Ahh, you're right, I just hate how verbose Java is and went with a more C#-esque style here to get around it 😅. But consistency is more important than my hangups with Java, so I'll update accordingly.

Reply:
Yeah, I feel that.  Modern C++ style also tends to drop the "gets" and "sets" nowadays.